doc/source/math/basic.rst

```diff
@@ -63,11 +63,22 @@ constants about the universe that you may find handy in your math operations.
 
 .. global:: Constant:PI
 
-    Ratio of circumference of a circle to its diameter
+    Ratio of circumference of a circle to its diameter, 3.14159265...
+    
+    ::
+
+        SET diameter to 10.
+        PRINT "circumference is:".
+        PRINT constant:pi * diameter.
```

Comment:
I'm with the “helpfulness for beginners” argument here, but mostly I thought it was weird that some of these had examples and some didn't. I don't think it hurts to have examples for each one.